schema/config-linux.json

```diff
@@ -170,11 +170,14 @@
                             },
                             "period": {
                                 "id": "https://opencontainers.org/schema/bundle/linux/resources/cpu/period",
-                                "$ref": "defs.json#/definitions/uint64Pointer"
+                                "type": "integer",
+                                "minimum": 1000,
+                                "maximum": 1000000
```

Comment:
Where do these magic numbers come from? If they come from the kernel, please document them in the `config-linux.md` file.